### ISSUE TITLE
[CY-4157] Released binary does not work

### ISSUE THREAD

**Issue by rtfpessoa** [codacy/codacy-faux-pas#10]
Stacktrace:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at scala.runtime.Statics$VM.mkHandle(Statics.java:172)
	at scala.runtime.Statics$VM.<clinit>(Statics.java:155)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:375)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:295)
	at scala.runtime.Statics.releaseFence(Statics.java:148)
	at scala.collection.immutable.$colon$colon.<init>(List.scala:586)
	at scala.collection.mutable.ListBuffer.addOne(ListBuffer.scala:108)
	at scala.collection.mutable.ListBuffer.addOne(ListBuffer.scala:38)
	at scala.collection.mutable.Growable.$plus$eq(Growable.scala:38)
	at scala.collection.mutable.Growable.$plus$eq$(Growable.scala:38)
	at scala.collection.mutable.AbstractBuffer.$plus$eq(Buffer.scala:231)
	at scopt.OptionParser.updateOption(options.scala:427)
	at scopt.OptionParser.makeDef(options.scala:423)
	at scopt.OptionParser.opt(options.scala:268)
	at scopt.OptionParser.opt(options.scala:275)
	at com.codacy.fauxpas.ParserConfig$$anon$1.<init>(ParserConfig.scala:12)
	at com.codacy.fauxpas.ParserConfig$.parse(ParserConfig.scala:11)
	at com.codacy.fauxpas.ParserConfig$.withConfig(ParserConfig.scala:20)
	at com.codacy.fauxpas.Main$.main(Main.scala:17)
	at com.codacy.fauxpas.Main.main(Main.scala)
Caused by: java.lang.NoSuchMethodException: no such method: java.lang.invoke.VarHandle.releaseFence()void/invokeStatic
	at java.lang.invoke.MemberName.makeAccessException(MemberName.java:963)
	at java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1101)
	at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:2030)
	at java.lang.invoke.MethodHandles$Lookup.findStatic(MethodHandles.java:1102)
	at scala.runtime.Statics$VM.mkHandle(Statics.java:161)
	... 19 more
Caused by: java.lang.NoSuchMethodError: java.lang.invoke.VarHandle.releaseFence()
	at java.lang.invoke.MethodHandleNatives.resolve(MethodHandleNatives.java:230)
	at java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1070)
	at java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1098)
	... 22 more
```

Possibly related: https://github.com/scala/bug/issues/12116

**Comment by rtfpessoa** [comment]
Also, since this is a tool for MacOS shouldn't the binary be built for it?

**Comment by josemiguelmelo** [comment]
The binary problem is fixed on version [0.1.4](https://github.com/codacy/codacy-faux-pas/releases/tag/0.1.4).

As the tool is for MacOS, it makes sense to have a binary for this OS. [This PR](https://github.com/codacy/codacy-faux-pas/pull/13) adds the binary generation for it and we are now publishing a version of the tool for darwing (starting on version [0.2.0](https://github.com/codacy/codacy-faux-pas/releases/tag/0.2.0))
